packages/core/admin/admin/src/hooks/useContentTypes/tests/useContentTypes.test.js

```diff
@@ -0,0 +1,144 @@
+import * as React from 'react';
+import { setupServer } from 'msw/node';
+import { rest } from 'msw';
+import { waitFor } from '@testing-library/react';
```

Comment:
I don't think you should mix these, `waitFor` is returned from the `render` function of `@testing-library/react-hooks` – wdyt?

Reply:
Fixed and rebased. Good catch - didn't know the render function returns `waitFor`.

Reply:
only in `react-hooks` lol